freeipa/src/main/java/com/sequenceiq/freeipa/converter/cloud/StackToCloudStackConverter.java

```diff
@@ -154,7 +154,7 @@ private CloudInstance buildInstance(Stack stack, InstanceMetaData instanceMetaDa
                 id,
                 instanceTemplate,
                 instanceAuthentication,
-                instanceMetaData.getSubnetId(),
+                instanceMetaData != null ? instanceMetaData.getSubnetId() : null,
```

Comment:
Perhaps add an UT case for this.

Reply:
this is coming from master. Probably some rebase issue